snafu/benchmarks/nighthawk/README.md

```diff
@@ -0,0 +1,46 @@
+# Steps to generate nighthawk-base image
+
+1. Create a ubuntu based pod using the below yaml
+    ```
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+    labels:
+        app: http-scale-client-test
+    name: http-scale-client-test
+    spec:
+    replicas: 1
+    selector:
+        matchLabels:
+        app: http-scale-client-test
+    template:
+        metadata:
+        labels:
+            app: http-scale-client-test
+        spec:
+        imagePullPolicy: Always
+        hostNetwork: true
+        tolerations:
+        - effect: NoSchedule
+            key: role
+            operator: Equal
+            value: workload
+        containers:
+        - name: ubuntu
+            image: ubuntu
+            command:
+            - sleep
+            - inf
+    ```
+2. Login to the host using ```oc rsh http-scale-client-test-7d45c599db-tdvn7 /bin/bash```
```

Comment:
I'd drop the ID from the login command since it wont be the same for anyone else. 
oc rsh http-scale-client-test-[ID] /bin/bash

Reply:
Changed it